mambaSharedFramework/Pantos-Generic Playlist Parsing/PantosTag.swift

```diff
@@ -417,14 +417,14 @@ extension PantosTag: HLSTagDescriptor, Equatable {
                 DictionaryTagValueIdentifierImpl(valueId: PantosValue.bandwidthBPS, optional: false, expectedType: Int.self),
                 DictionaryTagValueIdentifierImpl(valueId: PantosValue.programId, optional: true, expectedType: Int.self),
                 DictionaryTagValueIdentifierImpl(valueId: PantosValue.codecs, optional: true, expectedType: String.self),
-                DictionaryTagValueIdentifierImpl(valueId: PantosValue.resolution, optional: true, expectedType: HLSResolution.self),
+                DictionaryTagValueIdentifierImpl(valueId: PantosValue.resolution, optional: true, expectedType: ResolutionValueType.self),
                 DictionaryTagValueIdentifierImpl(valueId: PantosValue.uri, optional: false, expectedType: String.self),
                 DictionaryTagValueIdentifierImpl(valueId: PantosValue.videoGroup, optional: true, expectedType: String.self),
                 ])
             
         case .EXT_X_KEY:
             return EXT_X_KEYValidator(tag: pantostag, dictionaryValueIdentifiers: [
-                DictionaryTagValueIdentifierImpl(valueId: PantosValue.method, optional: true, expectedType: HLSEncryptionMethodType.self),
+                DictionaryTagValueIdentifierImpl(valueId: PantosValue.method, optional: true, expectedType: EncryptionMethodType.self),
```

Comment:
The same observation as the prior comment for the above 3 changes. I'm going to quit making these kinds of observations from here on.